lambda_function/aws/resources/acm_cert_handler.py

```diff
@@ -0,0 +1,80 @@
+import json
+import logging
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import boto3
+import consts
+from aws.resources.base_handler import BaseHandler
+from port.entities import create_entities_json, handle_entities
+
+logger = logging.getLogger(__name__)
+
+
+class ACMHandler(BaseHandler):
+    def handle(self):
+        for region in list(self.regions):
+            aws_acm_client = boto3.client("acm", region_name=region)
+            try:
+                paginator = aws_acm_client.get_paginator('list_certificates')
+                for page in paginator.paginate():
+                    certificate_summary_list = page.get('CertificateSummaryList', [])
+                    self._handle_list_response(certificate_summary_list, region)
+
+                    if self.lambda_context.get_remaining_time_in_millis() < consts.REMAINING_TIME_TO_REINVOKE_THRESHOLD:
+                        # Lambda timeout is too close, should return checkpoint for next run
+                        return self._handle_close_to_timeout(region)
+            except Exception as e:
+                logger.error(f"Failed to list ACM certificates in region: {region}; {e}")
+                self.skip_delete = True
+
+        return {
+            "aws_entities": self.aws_entities,
+            "next_resource_config": None,
+            "skip_delete": self.skip_delete,
+        }
+
+    def _handle_list_response(self, certificate_summary_list, region):
+        with ThreadPoolExecutor(max_workers=consts.MAX_CC_WORKERS) as executor:
+            futures = [
+                executor.submit(self.handle_single_resource_item, region, cert.get("CertificateArn", ""))
+                for cert in certificate_summary_list
+            ]
+            for completed_future in as_completed(futures):
+                result = completed_future.result()
+                self.aws_entities.update(result.get("aws_entities", set()))
+                self.skip_delete = result.get("skip_delete", False) if not self.skip_delete else self.skip_delete
+
+    def _handle_close_to_timeout(self, region):
+        if "selector" not in self.resource_config:
```

Comment:
Add here the relevant code to handle `next_token` and `self._cleanup_regions(region)`